rust/snippets/spice-rs/main.rs

```diff
@@ -1,11 +1,11 @@
 use futures::stream::StreamExt;
 use spice_rs::*;
 
-let mut spice_client = new_spice_client(api_key).await.expect("Failed to new spice client");
+let mut spice_client = new_spice_client(api_key).await;
 
 match spice_client.query(
-    "SELECT number, \"timestamp\", base_fee_per_gas, base_fee_per_gas / 1e9 AS base_fee_per_gas_gwei FROM eth.recent_blocks limit 10".to_string(),
-    None).await {
+    "SELECT number, \"timestamp\", base_fee_per_gas, base_fee_per_gas / 1e9 AS base_fee_per_gas_gwei FROM eth.recent_blocks limit 10".to_string()
```

Comment:
string literals aren't strings in rust?

Reply:
Only "" are strings in rs, so \ are needed for escaping "" around timestamp

Reply:
I'm referring to calling `to_string()` on a string literal. Why does the string literal need to be converted to a string?

Reply:
&str is the "" in Rust